### PR TITLE
some CSS added and icons

### PR DESCRIPTION
**Pull request by Zarzarius**
I added icons with React icons instead of texts in the buttons. I added a bit CSS to make the app looking more cool... Nothing definitive. I hope we can do something more together but I started because I am excited ;)

### CODE REVIEW

**Comment by AdrianValdes** [comment]
Por aquí hablamos ahora. Lo ves?

**Comment by Zarzarius** [comment]
Pues si , perfecto

**Comment by AdrianValdes** [comment]
Podrías añadir una descripción al commit? Puedes hacerlo desde Github. Click en los ... Donde dice: No description provided

**Comment by AdrianValdes** [comment]
Algo pequeño. Puede ser todo. Desde: No es mucho, solo icons y esto y lo otro. También cómo añadiste las cosas, o sea qué usaste para los íconos.

**Comment by Zarzarius** [comment]
Algo asi te refieres???? Me lio un poco aun con Github

**Comment by AdrianValdes** [comment]
Good for the comments. Could you remove the first person? So no:

> I added icons with React icons instead of texts in the buttons. 

Better: "Added icons with `react-icons`".

**Comment by AdrianValdes** [comment]
I am looking at what you did now, nice. It looks a lot better. And I also get confused, I am learning too! It is super crazy, I have to read everything 17 times. This is the FIRST time I am doing this review thing. I started to do what you are doing now last week. I mean, to be in the position of the person that asks for the review.

File: src/components/App.jsx [hunk 1]
```diff
@@ -21,10 +21,10 @@ function App() {
   };
 
   return (
-    <div>
-      <h1>Todo App</h1>
+    <div className="App" >
+      <h1 className= "Title">Todo App</h1>
```
Comment (AdrianValdes) [inline src/components/App.jsx]: why capital "T"? We should follow some kind of rule for the css class names, any idea?

File: src/components/Todo.jsx [hunk 2]
```diff
@@ -16,19 +18,19 @@ export default function Todo({
 
   if (isEditing) {
     return (
-      <div>
+      <div className= "editing">
```
Comment (AdrianValdes) [inline src/components/Todo.jsx]: Should we use nouns and not verbs for the classes? Editing could be very confusing, Maybe something like `edit-template-container`. But you think too, let's brain storm.

File: src/components/Todo.jsx [hunk 3]
```diff
@@ -47,9 +49,9 @@ export default function Todo({
         type="button"
         onClick={() => dispatch({ type: REMOVE_TODO, payload: { id } })}
       >
-        X
+       <RiDeleteBinFill />
       </button>
-      <button onClick={() => setIsEditing(true)}>Edit</button>
+      <button onClick={() => setIsEditing(true)}><RiEdit2Fill /></button>
```
Comment (AdrianValdes) [inline src/components/Todo.jsx]: Could we change the order of this icons. So right now you can see, from left to rigth, first the `delete icon` and then the `edit-icon`. I think that is a bit weird. I should be first edit and then delete. What do you think?

File: src/style/App.css [hunk 4]
```diff
@@ -1,8 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&family=Stylish&display=swap');
+
+*  {
+  
+  margin: 0;
```
Comment (AdrianValdes) [inline src/style/App.css]: Why this space here?

File: src/style/App.css [hunk 5]
```diff
@@ -1,8 +1,93 @@
...
+}
+.App {
+  margin: 3rem 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.Title {
+  font-family: 'Stylish',
+  sans-serif;
```
Comment (AdrianValdes) [inline src/style/App.css]: Could you review the format please? Sometimes you have spaces after each declaration and sometimes not, could we be consistent with just one? I like spaces, but you can decided.

File: src/style/App.css [hunk 6]
```diff
@@ -12,3 +97,25 @@ li {
 .priority-urgent {
   color: red;
 }
+
```
Comment (AdrianValdes) [inline src/style/App.css]: Could we change the colors for `important` and `urgent`? They don't look good I think. Maybe we can choose a "better" red and a "better" yellow?

File: src/style/App.css [hunk 7]
```diff
@@ -12,3 +97,25 @@ li {
...
+.App{
+  height: 100vh;
+}
+
+
```
Comment (AdrianValdes) [inline src/style/App.css]: This should go on the top.

File: src/style/App.css [hunk 8]
```diff
@@ -12,3 +97,25 @@ li {
...
+input[type="checkbox"],
+span,button {
+  border: none;
+  margin-right: 0.4rem;
+}
+
+span {
+  margin-right: 0.4rem;
+}
```
Comment (AdrianValdes) [inline src/style/App.css]: Where are we putting general styles, at the top or at the beginning?

**Review by AdrianValdes — CHANGES_REQUESTED** [review]

File: src/components/App.jsx (hunk 1 above)
Comment (Zarzarius) [reply src/components/App.jsx]: sure...I am mixing all the rules haha

File: src/components/Todo.jsx (hunk 2 above)
Comment (Zarzarius) [reply src/components/Todo.jsx]: good to know

**Comment by AdrianValdes** [comment]
You can always comment what I say, if you think it is necessary I mean.

File: src/components/Todo.jsx (hunk 3 above)
Comment (Zarzarius) [reply src/components/Todo.jsx]: done...you are right

File: src/style/App.css (hunk 5 above)
Comment (Zarzarius) [reply src/style/App.css]: done!

File: src/style/App.css (hunk 8 above)
Comment (Zarzarius) [reply src/style/App.css]: general styles you mean when is nothing with classes? like like targeting an html tag??

**Review by AdrianValdes — APPROVED** [review]
It looks good aza. Merge this one in!

**Comment by AdrianValdes** [comment]
Hi aza, you can merge this one in!

**Pull request merged**